microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -478,21 +498,34 @@ See also:
 * [Change colors with the Color Picker](../css/reference.md#change-colors-with-the-color-picker) in _CSS features reference_
 
 Status:
-*  This checkbox is present in Microsoft Edge Stable v103.
-*  This checkbox is present in Microsoft Edge Canary v105.
+*  This checkbox is present in Microsoft Edge Canary v106.
+*  This checkbox is present in Microsoft Edge Stable v104.
 
 
 <!-- ====================================================================== -->
 ## Group sources into Authored and Deployed trees
-<!-- new in v105 -->
+
+_new as of v105_
 
 Controls whether to group sources into **Authored** and **Deployed** trees in the **Page**, **Filesystem**, and **Overrides** tabs of the **Sources** tool.  This feature in the **Sources** tool allows you to group source files in two folders depending on whether these are:
```

Comment:
@captainbrosset - pls review/clarify desc

Reply:
This was added in 104, not 105. We talked about it in the What's New 104 video here https://www.youtube.com/watch?v=0_ZprFX8x0I&t=2s

```suggestion
Controls whether to group resources into separate **Authored** and **Deployed** trees in the **Page** tab of the **Sources** tool.  This feature in the **Sources** tool allows you to group source files in two folders depending on whether these are:
```